src/redux/build-data-actions.js

```diff
@@ -1,3 +1,4 @@
+import _ from 'lodash';
```

Comment:
I'd like to remove the need for lodash, ES6 and on gives us so much power that I think we can ween ourselves off of it.

Reply:
This the first time we're using this function with a nested state. We decided we'd move forward with this, but perhaps modify all code to pass an array of strings instead... perhaps renamed `keys` or something similar.